packages/lesswrong/server/callbacks/commentCallbacks.ts

```diff
@@ -440,13 +440,13 @@ getCollectionHooks("Comments").updateAfter.add(async function UpdateDescendentCo
 });
 
 // This function and the latter function seem redundant. TODO decide whether/where the karma < 100 clause should live
-getCollectionHooks("Comments").createAsync.add(async function NewCommentNeedsReview ({document}: CreateCallbackProperties<DbComment>) {
-  const user = await Users.findOne({_id:document.userId})
-  const karma = user?.karma || 0
-  if (karma < 100) {
-    await triggerReviewIfNeeded(document.userId);
-  }
-});
+// getCollectionHooks("Comments").createAsync.add(async function NewCommentNeedsReview ({document}: CreateCallbackProperties<DbComment>) {
+//   const user = await Users.findOne({_id:document.userId})
+//   const karma = user?.karma || 0
+//   if (karma < 100) {
+//     await triggerReviewIfNeeded(document.userId);
+//   }
+// });
```

Comment:
@Raemon Since this is redundant with the one below, I just commented it out. I'll leave the code here so you can decide whether to incorporate or delete this later.